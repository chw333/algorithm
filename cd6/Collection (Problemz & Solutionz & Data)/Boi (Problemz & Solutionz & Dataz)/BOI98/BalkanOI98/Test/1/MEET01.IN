108 3 33
10 1 5
12 1 2
13 7 2
14 6 6
17 11 7
20 6 7
21 19 4
23 3 8
26 20 6
27 19 6
27 26 5
28 14 4
28 26 7
30 5 2
30 29 7
31 11 1
31 18 4
31 21 4
32 1 8
37 3 5
37 6 2
37 15 2
37 33 7
38 1 8
40 17 5
40 27 7
41 4 2
41 15 6
43 27 8
44 1 6
44 22 8
44 30 3
45 1 10
45 9 4
45 16 2
46 1 3
47 1 7
48 7 7
48 34 3
48 39 4
49 8 8
49 26 2
49 28 5
51 9 8
52 3 2
52 29 7
52 33 7
53 33 6
54 1 4
56 41 3
57 5 6
59 26 9
60 26 7
61 13 2
63 1 3
64 41 4
64 52 4
65 37 5
65 58 7
66 34 9
68 36 3
68 65 6
70 30 7
71 41 7
71 58 3
73 1 7
74 7 8
75 1 8
75 50 3
76 57 1
77 1 2
78 2 9
78 36 4
79 4 4
79 8 9
79 22 8
79 40 3
80 21 9
80 35 6
80 70 4
81 29 6
82 3 9
82 33 8
82 65 7
82 69 5
82 74 7
83 11 2
84 19 4
84 23 7
84 72 5
85 59 3
86 13 3
86 25 10
86 29 10
86 42 7
86 59 4
87 17 7
87 62 9
88 2 2
88 15 4
88 55 7
88 67 2
89 15 3
89 24 5
89 28 5
89 34 8
89 83 2
90 1 9
