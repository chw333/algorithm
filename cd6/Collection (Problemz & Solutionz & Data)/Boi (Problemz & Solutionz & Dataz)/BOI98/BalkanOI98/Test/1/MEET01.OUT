3 82 3 37
33 52 64 41 15 37
